x/group/keeper/invariants.go

```diff
@@ -38,12 +38,8 @@ func GroupTotalWeightInvariantHelper(ctx sdk.Context, key storetypes.StoreKey, g
 	}
 	defer groupIt.Close()
 
+	groups := make(map[uint64]group.GroupInfo)
 	for {
-		membersWeight, err := groupmath.NewNonNegativeDecFromString("0")
```

Comment:
was this line failing under some conditions? I'm not understanding how the behavior of parsing `"0"` into a `Dec` could be anything but invariant

Reply:
this has nothing to do with the fix/changes. The change was to not have nested iterators which a violation of how iterators should be used.